call-home/src/collector/report_models.rs

```diff
@@ -85,34 +77,29 @@ impl Pools {
 
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(rename_all = "camelCase")]
-pub struct Replicas
-{
+pub struct Replicas {
     count: u64,
     count_per_volume_percentiles: Percentiles,
 }
 impl Replicas {
-    ///Returns a replica object with default values
-    pub fn default() -> Self
-    {
+    /// Returns a replica object with default values
+    pub fn default() -> Self {
```

Comment:
default should be using Default trait, and can be automatically added simply by adding Default to the derive.